src/OpenTelemetry.Exporter.Prometheus.AspNetCore/PrometheusAspNetCoreOptions.cs

```diff
@@ -17,6 +17,15 @@ public class PrometheusAspNetCoreOptions
     /// </summary>
     public string ScrapeEndpointPath { get; set; } = DefaultScrapeEndpointPath;
 
+    /// <summary>
+    /// Gets or sets a value indicating whether addition of _total suffix for counter metric names is disabled. Default value: <see langword="false"/>.
+    /// </summary>
+    public bool DisableTotalNameSuffixForCounters
```

Comment:
`public bool EnableCounterTotalSuffix = true` - alternate suggestion.

The default should be true as per https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/compatibility/prometheus_and_openmetrics.md#sums

Reply:
> Exporters SHOULD provide a configuration option to disable..

So I think _DisableTotalNameSuffixForCounters_ will be better